railties/test/generators/app_generator_test.rb

```diff
@@ -686,6 +686,19 @@ def test_after_bundle_callback
     end
   end
 
+  def test_dotrails_file_present
+    run_generator
+
+    assert_file ".rails"
```

Comment:
We should also test the contents of the file. Let's run the generator with some options and see what ends up in the file.